scripts/indexer-schema/generate.sh

```diff
@@ -63,5 +63,7 @@ docker exec ${CONTAINER_NAME} diesel print-schema \
   --except-tables "^objects_version_|_partition_" \
   > "${REPO}/crates/iota-indexer/src/schema.rs"
 
+python3 ${REPO}/scripts/indexer-schema/generate_for_all_tables_macro.py "${REPO}/crates/iota-indexer/src/schema.rs"
```

Comment:
You can't know if the system uses `python` or `python3` as the command. That's why I created the `python_venv_wrapper.sh`, which will set an environment variable `PYTHON_CMD` for you, and also execute the script in a virtual environment. Maybe you want to use that instead.

Example can be found in the `run.sh` scripts in the scripts folder.

Reply:
We don't need venv logic here, so decided to just use the cmd selection part.
If you don't like the copypaste we could extract this part to a separate script I guess.